packages/grid/_modules_/grid/hooks/utils/useGridRootProps.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { GridRootPropsContext } from '../../context/GridRootPropsContext';
+
+export const useGridRootProps = () => {
```

Comment:
I think that is better. I added @m2mathew but other than that I'm ok with the change.